docs/usage.md

```diff
@@ -0,0 +1,54 @@
+# Migration Tool Usage Instruction and Examples
+
+This is the complete utilization guide for this tool, these next sections will explain how this application works and how users can do a complete HTML migration to XML migration to the new Publishing Platform.
+
+## Topics
+1. Obtaining HTML documents
+2. Converting HTML to XML documents
+3. [Updating documents' mixed citations](#3-\--updating-documents-mixed-citations)
+4. Generating documents packages
+5. Importing documents packages to the Publish Platform
+6. Committing the relationship between documents and issues
+
+## 3 - Updating documents' mixed citations
+
+The documents must have their mixed citations updated, its important to preserve the quality of your collection. The information stored in the articles' database or paragraphs' databases is essential to finish this task, then please make sure you have to access these databases.
```

Comment:
Corrigir: "make sure you have to access these databases." 

"make sure you have access to these databases.

Reply:
Erro no engano, valeu @robertatakenaka.